src/System.Windows.Forms/src/System/Windows/Forms/ComponentModel/COM2Interop/COM2Enum.cs

```diff
@@ -113,7 +111,7 @@ public virtual string ToString(object value)
 
             // We have to do this to compensate for small discrepancies in a lot of objects in COM2
             // (DWORD -> VT_IU4, value we get is VT_I4, which convert to Int32, UInt32 respectively)
-            string stringValue = value.ToString();
+            string? stringValue = value?.ToString();
```

Comment:
I don't think this correctly expresses the intent - i.e. an enum is not a nullable structure. Whilst we're operating with objects here, we never expect a `null` as a result. In an unlikely event when we receive a `null`, we should just return
```suggestion
            if (value?.ToString() is not string stringValue)
            {
                return string.Empty;
            }
```